packages/react/react/src/utils/convertNewlineToJSX.tsx

```diff
@@ -7,7 +7,7 @@ export function convertNewlineToJSX(str: string) {
 
   return chunks.map((line, index) => (
     <Fragment key={index}>
-      {index > 0 ? <br /> : ''}
+      {index > 0 && <br />}
```

Comment:
I thought it would be better to write the code more cleanly using the && operator, rather than including unnecessary ''. how about like this?

Reply:
@okinawaa I changed it to return null instead of an empty string. I think this way readability improves, and it helps avoid returning unnecessary things. Could you check if it [works](https://github.com/toss/slash/pull/350/commits/14ff32a3d3657f1b9062a33aeae3c2d767c0e303)?